assets/dev/js/editor/elements/views/base.js

```diff
@@ -645,6 +652,13 @@ BaseElementView = BaseContainer.extend( {
 				}
 			} );
 
+
+			if ( isAtomicRepeater ) {
+				dynamicValue = this.getDynamicValue( settings );
+
+				this.renderDynamicValue( dynamicValue );
```

Comment:
I am not sure that this is the best place for this function, 
Maybe it should be inside one of the render functions (renderHTML or render UI)